documentation/installation.md

```diff
@@ -49,16 +60,24 @@ command:
 
     vagrant ssh
 
-To check if everything is set up properly you can run ``make test.unit`` inside
+Inside the virtual machine, if you are setting up a development environment,
```

Comment:
@alanoe, the vagrant VM and the playbooks that run on it, in particular, are only for setting up a development environment.

Reply:
Done